src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.server.ts

```diff
@@ -45,8 +43,18 @@ const query = gql`
         subheading
         description {
           json
+          links {
+            assets {
+              block {
+                ...ImageProps
+              }
+              hyperlink {
+                ...ImageProps
+              }
+            }
+          }
         }
-        serviceEntriesCollection(limit: 10) {
+        serviceEntriesCollection(limit: 5) {
```

Comment:
You might have been hitting query complexity limits, but this'll be too small for some service entries. E.g. `/animals/meat-poultry` currently has 8 service entries, so we'll probably want at that. Ideally figure out with UX what the ceiling on number of service entries might be (10? higher?)